src/activities/ScoringEntity.js

```diff
@@ -49,7 +49,7 @@ export class ScoringEntity extends Entity {
 		}
 
 		if (scoring.gradeMaxPoints !== this.gradeMaxPoints()) {
-			const fields = [{ name: this._getUpdateFieldName(), value: scoring.gradeMaxPoints }];
+			const fields = [{ name: this._getUpdateFieldName(), value: scoring.gradeMaxPoints || '' }];
```

Comment:
Was `undefined` or `null` being set for no score?

Reply:
both, depending on where (it was null initially when you clicked ungraded button, and undefined if you cleared the input manually).